validation/longhorn/schemas/pit_schemas.md

```diff
@@ -0,0 +1,399 @@
+# RANCHERINT Schemas
+
+## Test Suite: Longhorn
+
+### Fresh Longhorn Installation via Rancher App Catalog
+
+Verify Longhorn can be successfully installed through Rancher's app catalog with default settings on a clean Kubernetes cluster.
```

Comment:
since none of these are automated, can you update this to `TBD`? Or better yet, enhance the main.go file for qase once again to be `contains` instead of `equals`, that way you can keep these nice descriptions https://github.com/slickwarren/tests/blob/main/validation/pipeline/qase/schemaupload/main.go#L113